Source/common/SNTFileInfo.m

```diff
@@ -78,33 +74,17 @@ - (instancetype)initWithPath:(NSString *)path
       return nil;
```

Comment:
I would change this error handling to be a bit clearer - this error suggests there was an error resolving the path but this is in the resolvedPath method, so this error should move to initWithPath (which will mean splitting out the path resolving from the init. This error should then call out the fact the path was empty.